cmd/kubeadm/app/constants/constants.go

```diff
@@ -65,6 +65,11 @@ const (
 	// APIServerKubeletClientCertCommonName defines kubelet client certificate common name (CN)
 	APIServerKubeletClientCertCommonName = "kube-apiserver-kubelet-client"
 
+	// AuditPolicyFileName defines audit policy resource file for apiserver
+	AuditPolicyFileName = "audit.yaml"
+	// AuditPolicyLogFileName defines the audit log filename
+	AuditPolicyLogFileName = "audit.log"
```

Comment:
Audit log uses lumberjack under the hood which uses timestamps embedded in the filename to determine age of logs. I don't think defining an AuditPolicyLogFileName is the right thing to do here, unless I'm missing something.

Reply:
nope, my misundertsanding. lumberjack will take care of this automatically after the max-size.